db/migrate/20190807074256_create_musician_skills_users.rb

```diff
@@ -0,0 +1,9 @@
+#:reek:FeatureEnvy and :reek:UncommunicativeVariableName
+class CreateMusicianSkillsUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :musician_skills_users do |t|
```

Comment:
This is connection table and I guess you need `user_id` and `musician_skill_id` in here

Reply:
What for do you need `state` in here?